geomesa-geojson/geomesa-geojson-rest/pom.xml

```diff
@@ -58,6 +62,15 @@
             <groupId>org.scalatra</groupId>
             <artifactId>scalatra-specs2_${scala.binary.version}</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-accumulo-datastore_${scala.binary.version}</artifactId>
+            <classifier>tests</classifier>
```

Comment:
The same dependency is listed twice.  Likely with two different scopes.

Reply:
this is the 'tests' classifier which brings in the accumulo minicluster wrapper and test classes.

Reply:
*facepalm*